### PR TITLE
fix scheduled download tests

### PR DESCRIPTION
**Pull request by pmeier**
I've messed up the syntax for the workflow triggers in #2675. Thus, the CI does not even spin up. I think, it uses `push` as a default trigger, which is why we see [some failed runs](https://github.com/pytorch/vision/actions).

---

Edit: Unfortunately, I need to fix two more things:

- [x] Using an abstract base for the download tester is not working as expected: the abstract class is also run which expectedly fails, since the abstract methods are not implemented. Skipping this also skips all subclasses and thus no test is run. I've rewritten the collection logic to reflect that.
- [ ] ~The issue generation is bugged for labels including a colon such as `module: datasets`. I've [opened and issue](https://github.com/JasonEtco/create-an-issue/issues/65). Depending on the response speed of the maintainer we either need to simply upgrade the action version or remove the label.~

### CODE REVIEW

File: test/test_datasets_download.py [hunk 1]
```diff
@@ -43,89 +44,94 @@ def inner_wrapper(request, *args, **kwargs):
...
+    argvalues = []
+    ids = []
+    for config in download_configs:
+        argvalues.append((config.url, config.md5))
+        ids.append(config.id)
+
+    return dict(argnames="url, md5", argvalues=argvalues, ids=ids)
+
+
+def places365():
```
Comment (pmeier) [inline test/test_datasets_download.py:119]: To add new download tests for a dataset, simply implement a similar function and add it to L130 and optionally to L135.

**Comment by pmeier** [comment]
As reported in https://github.com/JasonEtco/create-an-issue/issues/65#issuecomment-698794002 this issue most likely does not lie in the action itself, but rather in a dependency. Thus, the fix is probably going to take a while and I've implemented a workaround. This means that the automatically created issue will not be labeled `module: datasets` for now, but I've assigned myself anyway.

**Comment by pmeier** [comment]
After some further inspection the issue does not lie with https://github.com/JasonEtco/create-an-issue, but rather with our repository configuration. I suspect the GitHub Actions bot has no permission to open the issue.

As a workaround, I'm disabling the issue generation for now and re-enable it as soon as this is resolved. IMO we should merge this now to at least have the test although their success / failure has to be checked manually. Since I'm not able to debug this any further myself, I can't tell how long it takes until this will be completely resolved.

**Comment by pmeier** [comment]
@fmassa Test failure is unrelated.

**Comment by pmeier** [comment]
#2714 showed that the issue creation works as expected when performed from a scheduled workflow. Thus, the last commit re-enables it.

**Review by fmassa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**